### ISSUE TITLE
Multiple plots per frame

### ISSUE THREAD

**Issue by agarie** [domitry/nyaplot#35]
I searched the documentation for some example, but I couldn't find how to generate something similar to the subplots of MATLAB or Matplotlib.

Is it possible right now? If not, this would be really useful.

**Comment by domitry** [comment]
Current version of Nyaplot can generate subplots arranged vertically.
http://nbviewer.ipython.org/github/domitry/nyaplot/blob/master/examples/notebook/Interaction_with_DataFrame.ipynb

Ones arranged horizontally are available with Nyaplot v0.2.x, but it is still in rc1 version.
```ruby
include Nyaplot
p1 = Plot.from(df).add(:scatter, x: :x, y: :y)
p2 = Plot.from(df).add(:histogram, value: :z)
columns(p1, p2)
```

**Comment by agarie** [comment]
Thanks, @domitry! I'm currently working on improving the tutorials (examples/notebook) based on what I felt using Nyaplot and the feedback from some coworkers and friends that tried it.
